numba/cuda/tests/cudapy/test_intrinsics.py

```diff
@@ -292,6 +341,150 @@ def test_fma_f8(self):
         compiled[1, 1](ary, 2., 3., 4.)
         np.testing.assert_allclose(ary[0], 2 * 3 + 4)
 
+    def toolkit_and_compute_supported(self, compute_major, compute_minor,
+                                      toolkit_major, toolkit_minor):
```

Comment:
Perhaps the interface could be:
```python
def toolkit_and_compute_supported(compute=(major, minor), toolkit=(major, minor))
```
so as to make it easier to identify which values belong to which requirement?

Reply:
Or even two decorators, `toolkit_support(major, minor)` and `compute_support(major, minor)`. CC @gmarkall as code owner.

Reply:
I think separate functions `toolkit_support` and `compute_support` would be easier to read at the call site so I'd prefer that option.